public/app/features/plugins/components/PluginsErrorsInfo.tsx

```diff
@@ -17,35 +21,36 @@ export function PluginsErrorsInfo() {
   }
 
   return (
-    <InfoBox
+    <Alert
+      title="Unsigned plugins were found during plugin initialization. Grafana Labs cannot guarantee the integrity of these plugins. We recommend only using signed plugins."
```

Comment:
Probably unrelated to this PR, but are we sure that these plugin errors are always related to plugin signing? Could it be that a plugin is disabled because it's backend counterpart errored out?

Reply:
Hmm... looking at the [frontend](https://github.com/grafana/grafana/blob/jackw/filter-plugin-errors/packages/grafana-data/src/types/plugin.ts#L42-L53) and [backend](https://github.com/grafana/grafana/blob/jackw/filter-plugin-errors/pkg/plugins/models.go#L62-L75) it lookslike it is only plugin signature errors that are reported right now.